src/Views/Form/Templates/Sequoia/views/receipt.php

```diff
@@ -35,15 +35,46 @@
 		<?php if ( isset( $options['thank-you']['sharing'] ) && $options['thank-you']['sharing'] === 'enabled' ) : ?>
 			<div class="social-sharing">
 				<p class="instruction">
-					<?php __( 'Tell the world about your generosity and help spread the word!', 'give' ); ?>
+					<?php echo $options['thank-you']['sharing_instruction']; ?>
 				</p>
 				<div class="btn-row">
-					<button class="give-btn social-btn facebook-btn">
+					<!-- Use inline onclick listener to avoid popup blockers -->
+					<button class="give-btn social-btn facebook-btn"
+						onclick="
+							// Retrieve and sanitize url to be shared
+							let url = parent.window.location.toString();
+							if (url.includes('?giveDonationAction=showReceipt')) {
```

Comment:
@henryholtgeerts use `Give.fn.getParameterByName` to get data by param key

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/1a56a5bf2e33f0ca14064c39fa4021d03590864f